src/loaders/svg/tvgSvgLoader.cpp

```diff
@@ -780,6 +780,7 @@ static void _handlePaintAttr(SvgPaint* paint, const char* value)
 static void _handleColorAttr(TVG_UNUSED SvgLoaderData* loader, SvgNode* node, const char* value)
 {
     SvgStyleProperty* style = node->style;
+    style->curColorSet = true;
```

Comment:
I think `curColorSet` should be initialize `false` in `_createNode()`.
+) And the curColor variable is also missing initialization. I recommend to initialize them together.

Reply:
Thx! 
I remembered about that, until I forgot... :D